dts/arc/tenstorrent/tt_blackhole_smc.dtsi

```diff
@@ -312,6 +312,11 @@
 		status = "disabled";
 	};
 
+	noc_dma: noc_dma@1234000 {
```

Comment:
It might be good to reserve the ARC HS DMA as `dma0`, and then use `dma1`, `dma2` for NOC DMA instances. I'm not sure if we ever use the second NOC from ARC though, so maybe it doesn't make sense to add `dma2`.

```suggestion
	dma1: noc_dma@1234000 {
```